git.go

```diff
@@ -33,7 +33,9 @@ func getBranches(all_branches bool) []string {
 		branches = strings.Split(branches_str, "\n")
 
 		// Branches list is separated by spaces. Let's put it in an array
-		labels = append(labels, branches...)
+		if !strings.Contains(branches_str, "tags") {
```

Comment:
If I create a branch named "catagsarm" it will be omitted this way because it contains 'tags' inside the text.

What is your current use-case? Does your git spit your tags as branches?